pkg/routeagent_driver/handlers/ovn/handler.go

```diff
@@ -70,9 +70,9 @@ type Handler struct {
 
 var logger = log.Logger{Logger: logf.Log.WithName("OVN")}
 
-func NewHandler(config *HandlerConfig) *Handler {
+func NewHandler(ipFamily k8snet.IPFamily, config *HandlerConfig) *Handler {
```

Comment:
Maybe we can save ipFamily in handler struct, so we can use it in rest of handler code